docs/man/containerd-config.toml.5.md

```diff
@@ -209,6 +226,12 @@ imports = ["/etc/containerd/runtime_*.toml", "./debug.toml"]
   [plugins."io.containerd.service.v1.tasks-service"]
     blockio_config_file = ""
     rdt_config_file = ""
+  [plugins.'io.containerd.image-verifier.v1.bindir']
+    bin_dir = '/opt/containerd/image-verifier/bin'
+    max_verifiers = 10
+    per_verifier_timeout = '10s'
+  [plugins.'io.containerd.monitor.container.v1.restart']
```

Comment:
Comparing 1.0 and 2.0 using vscode compare
<img width="1066" alt="Screenshot 2024-11-01 at 8 38 34 AM" src="https://github.com/user-attachments/assets/d0afc376-6445-43d1-bd9b-46e144ab73b2">